### PR TITLE
Add stereo image view funciton

### PR DESCRIPTION
**Pull request by ishiguroJSK**
I Implemented functions for stereo vision for HTC Vive
I copied and include hellovr_opengl_main.cpp from openvr's sample codes,
and inherited it's openGL visualization class.

Now, we can use most of all functions of HTC Vive in ROS.
```
roslaunch vive_ros vive.launch
rosrun usb_cam usb_cam_node _video_device:=/dev/videoXX
```

vive_ros_image

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -33,13 +63,24 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 ")
...
+  ${OPENVR_SAMPLE_UTILS}
+)
+target_link_libraries(vive_node
+  ${catkin_LIBRARIES}
+  ${OPENVR_LIB}
+  openvr_api
+  ${GLEW_LIBRARIES}
+  ${GTK2_LIBRARIES}
+  ${SDL2_LIBRARY}
+  ${OpenCV_LIBS}
```
Comment (k-okada) [inline CMakeLists.txt]: if you add `find_package(catkin REQUIRED cv_bridge)` then opencv libraries should provided by `${catkin_LIBRARIES}` , please check that

File: CMakeLists.txt [hunk 2]
```diff
@@ -7,20 +7,50 @@ find_package(catkin REQUIRED COMPONENTS
...
+message(STATUS "SDL2_INCLUDE_DIR '${SDL2_INCLUDE_DIR}' ")
+
+find_package(GLEW REQUIRED)
+include_directories(${GLEW_INCLUDE_DIRS})
+	
+find_package(GTK2)
+add_definitions(-DHAVE_GTK)
+include_directories(${GTK2_INCLUDE_DIRS})
+
+# find_package(OpenCV REQUIRED)
```
Comment (k-okada) [inline CMakeLists.txt]: remove this lines

File: CMakeLists.txt [hunk 3]
```diff
@@ -7,20 +7,50 @@ find_package(catkin REQUIRED COMPONENTS
...
+# include_directories(${OpenCV_INCLUDE_DIRS})
 
 
 ###################################
 ## catkin specific configuration ##
 ###################################
-catkin_package()
+#catkin_package()
```
Comment (k-okada) [inline CMakeLists.txt:44]: remove this comment line too

File: CMakeLists.txt [hunk 4]
```diff
@@ -33,13 +63,24 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 ")
 include_directories(
   include
   ${catkin_INCLUDE_DIRS}
-  ${OPENVR}/headers
```
Comment (k-okada) [inline CMakeLists.txt:36]: may be you do not need to remove existing lines, unless that's the cause of any trouble

File: htc_vive.rc [hunk 5]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (k-okada) [inline htc_vive.rc:1]: what is this file?

File: launch/vive.launch [hunk 6]
```diff
@@ -1,17 +1,34 @@
...
 
   <rosparam param="/vive/world_offset">[0, 0, 2]</rosparam>
   <rosparam param="/vive/world_yaw">0.0</rosparam>
   
+  <!-- <remap from="image_left" to="/multisense_local/left/image_rect_color"/> -->
+  <!-- <remap from="image_right" to="/multisense_local/right/image_rect"/> -->
+  <!-- <remap from="camera_info_left" to="/multisense_local/left/camera_info"/> -->
+  <!-- <remap from="camera_info_right" to="/multisense_local/right/camera_info"/> -->
+  
+  <remap from="image_left" to="/usb_cam/image_raw"/>
```
Comment (k-okada) [inline launch/vive.launch]: use CAMERA arg

File: CMakeLists.txt (hunk 2 above)
Comment (ishiguroJSK) [reply CMakeLists.txt]: OK

File: CMakeLists.txt (hunk 3 above)
Comment (ishiguroJSK) [reply CMakeLists.txt:44]: OK

File: CMakeLists.txt (hunk 4 above)
Comment (ishiguroJSK) [reply CMakeLists.txt:36]: I think I already added OpenVR/headers by `include_directories("${OPENVR}/headers" "${OPENVR}/samples")` above.
Is it correct way?

File: CMakeLists.txt (hunk 1 above)
Comment (ishiguroJSK) [reply CMakeLists.txt]: I see.
OK

File: htc_vive.rc (hunk 5 above)
Comment (ishiguroJSK) [reply htc_vive.rc:1]: This file is an optional file for sourcing SteamVR runtime path.
We need these additional libraries path when we use HTC VIVE.
Same descriptions are also written in the launch file so this file is optional.

File: launch/vive.launch (hunk 6 above)
Comment (ishiguroJSK) [reply launch/vive.launch]: I see.
But even between usb_cam and multisense, the image topic field names are different (image_raw and image_rect_color)
Is it really useful that introducing CAMERA arg ?

**Pull request closed without merging**